### PR TITLE
Add heartbeat to fis sdk

### PR DESCRIPTION
**Pull request by VinayGuthal**

### CODE REVIEW

File: firebase-installations/src/main/java/com/google/firebase/installations/remote/FirebaseInstallationServiceClient.java [hunk 1]
```diff
@@ -55,6 +59,12 @@
   private static final String CONTENT_ENCODING_HEADER_KEY = "Content-Encoding";
   private static final String GZIP_CONTENT_ENCODING = "gzip";
 
+  /** Heartbeat tag for firebase installations. */
+  private static final String FIREBASE_IID_HEARTBEAT_TAG = "fire-iid";
```
Comment (ankitaj224) [inline firebase-installations/src/main/java/com/google/firebase/installations/remote/FirebaseInstallationServiceClient.java]: FIREBASE_INSTALLATIONS_ID_HEARTBEAT_TAG
firebase-installations-id??

File: tools/measurement/apksize/src/inappmessagingdisplay/java/com.google.apksize/InAppMessagingDisplay.java [hunk 2]
```diff
@@ -16,16 +16,16 @@
 
 import android.app.Activity;
 import android.content.Context;
+import androidx.annotation.NonNull;
 import com.google.android.gms.tasks.Task;
 import com.google.android.gms.tasks.TaskCompletionSource;
 import com.google.firebase.inappmessaging.FirebaseInAppMessagingDisplayCallbacks;
+import com.google.firebase.inappmessaging.display.FirebaseInAppMessagingDisplay;
```
Comment (ankitaj224) [inline tools/measurement/apksize/src/inappmessagingdisplay/java/com.google.apksize/InAppMessagingDisplay.java]: These changes are irrelevant to FIS. Please verify.

File: firebase-installations/src/main/java/com/google/firebase/installations/remote/FirebaseInstallationServiceClient.java (hunk 1 above)
Comment (VinayGuthal) [reply firebase-installations/src/main/java/com/google/firebase/installations/remote/FirebaseInstallationServiceClient.java]: Great catch! Thanks

File: tools/measurement/apksize/src/inappmessagingdisplay/java/com.google.apksize/InAppMessagingDisplay.java (hunk 2 above)
Comment (VinayGuthal) [reply tools/measurement/apksize/src/inappmessagingdisplay/java/com.google.apksize/InAppMessagingDisplay.java]: This is just a result of google java format

**Pull request merged**